src/tracks/playabletrack/wavetrack/ui/WaveTrackAffordanceControls.cpp

```diff
@@ -228,6 +231,54 @@ unsigned WaveTrackAffordanceControls::KeyDown(wxKeyEvent& event, ViewInfo& viewI
     return RefreshCode::RefreshCell;
 }
 
+namespace {
+    template<typename Iter, typename Comp>
+    WaveClip* NextClipLooped(ViewInfo& viewInfo, Iter begin, Iter end, Comp comp)
```

Comment:
Nice use of a template to capture the common logic of the forward and backward cases!

Instead of `Comp comp` I would write `Pred pred` because that argument is a unary predicate, not a binary comparator.  But that's only variable names.